php/Terminus/Models/TerminusModel.php

```diff
@@ -6,25 +6,25 @@
 
 abstract class TerminusModel
 {
-  /**
-   * @var string
-   */
+    /**
+     * @var string
+     */
     public $id;
-  /**
-   * @var array Arguments for fetching this model's information
-   */
+    /**
+     * @var Request
+     */
+    public $request;
```

Comment:
Yeah, ideally Models would be injected with RESTful datasource adapter but that's a bigger task. Perhaps `$__request` would belay the intent to keep this internal enough for now.